sql/core/src/main/scala/org/apache/spark/sql/parquet/newParquet.scala

```diff
@@ -106,12 +106,12 @@ class DefaultSource
       ParquetRelation.createEmpty(
```

Comment:
Currently we are still using some utility functions like this one from the old Parquet support code. We can move them into the new data source in the future.